tests/manager/api/axis/test_api.py

```diff
@@ -362,118 +318,320 @@ def test_fulfill(self, axis360: Axis360Fixture):
 
         # If Axis 360 says a patron does not have a title checked out,
         # an attempt to fulfill that title will fail with NoActiveLoan.
-        data = axis360.sample_data("availability_with_audiobook_fulfillment.xml")
-        axis360.api.queue_response(200, content=data)
-        pytest.raises(NoActiveLoan, fulfill)
+        data = axis360.files.sample_data("availability_with_ebook_fulfillment.xml")
```

Comment:
Added better coverage for our fulfillment operations. Some of this comes from our old tests for parsers and fulfillment, and some of it is new tests.